.travis.yml

```diff
@@ -2,13 +2,20 @@ language: bash
 services: docker
 
 env:
-  image=eclecticiq/package
-
+  global:
+    image=eclecticiq/package
+    latest=1.11.0
+  matrix:
+    - version=1.11.0
```

Comment:
what about changing this to `- version=1.11.0 latest=true`? then you can drop the `latest=1.11.0`